### DISCUSSION TITLE
Welcome to ImportExportProfiles Discussions!

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/ImportExportProfiles#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by optophon** [comment]
importexportprofiles does not work under cura 5.3/5.4 on Mac os 12.6.7. 

greetings

optophon

**Comment by 5axes** [comment]
Sorry cannot reproduce on Windows
